### ISSUE TITLE
Update feature extraction to handle inner/outer pixels

### ISSUE THREAD

**Issue by lewfish** [azavea/raster-vision#603]
After #602 is done, update the feature extraction functionality to utilize inner / outer pixels in the predicted segmentation mask. The original functionality (which worked with just background and inner pixels) should still work.

**Comment by lewfish** [comment]
I'm not exactly sure how the feature extraction works, but I imagine it might be difficult to make it work with more than one object class (ie. a background, inner, and outer class id). If that's the case, you could spin that into a separate issue which has lower priority, since we just need this to work for buildings for now.

**Comment by lewfish** [comment]
I think this will be handled automatically given the way feature extraction works.
